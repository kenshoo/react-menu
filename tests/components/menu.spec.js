import React from "react";
import renderer from "react-test-renderer";
import Menu from "../../src/components/menu";

describe("Menu component", () => {
  test("renders children", () => {
    const cmp = renderer.create(
      <Menu>
        <div>TEST</div>
      </Menu>
    );
    expect(cmp).toMatchSnapshot();
  });

  test("recieves className", () => {
    const cmp = renderer.create(
      <Menu className="TEST_CLASS">
        <div>TEST</div>
      </Menu>
    );
    expect(cmp).toMatchSnapshot();
  });
});
