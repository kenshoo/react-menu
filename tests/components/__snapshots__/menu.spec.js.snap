// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component recieves className 1`] = `
<div
  className="menu TEST_CLASS"
>
  TEST
</div>
`;

exports[`Menu component renders children 1`] = `
<div
  className="menu"
>
  TEST
</div>
`;
